config.go

```diff
@@ -388,7 +388,7 @@ func DefaultConfig() Config {
 		Alias:                         defaultAlias,
 		Color:                         defaultColor,
 		MinChanSize:                   int64(minChanFundingSize),
-		MaxChanSize:                   int64(MaxFundingAmount),
+		MaxChanSize:                   int64(0), // Do not enforce maxiumum channel size by default
```

Comment:
I think we _do_ want to enforce it by default, but only when we're in wumbo mode. A default wumbo soft-cap of 10 BTC has been suggested.